packages/docusaurus-plugin-content-docs/src/props.ts

```diff
@@ -39,6 +39,7 @@ Available document ids=
       type: 'link',
       label: sidebar_label || title,
       href: permalink,
+      extra: item.extra,
```

Comment:
Personally, I like "data" more for the name of this attribute, but no strong opinion

@ReginaLiang what's your opinion?

Reply:
@slorber, 
Actually, I think any one will be fine. ^v^, a little little thought, How about `customFields`?  It is consistent with DocusaurusConfig object,  I guess `customFields` more meaningful. But any one is OK for me.

@slorber, @oriooctopus ,
Thanks for your guys great work! I believe docusaurus is getting better and better.